packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -48,7 +48,11 @@
 
     if (params.containsKey("initialUrl")) {
       String url = (String) params.get("initialUrl");
-      webView.loadUrl(url);
+      if (url.contains("://")) {
```

Comment:
This makes it so we accept something that isn't a URL as the `initialUrl`.
We could consider adding an `initialAsset` parameter, but I'd leave it for a different PR as it warrants some more discussion (generally it seems like users tend to get confused by the `initialFoo` parameters)